### ISSUE TITLE
Allow Flip Screen option for analog output (CRT)

### ISSUE THREAD

**Issue by capitaineflam25** [MiSTer-devel/Arcade-TimePilot_MiSTer#19]
Thanks for this great core.

Sometime your vertical CRT is not orientated the right way and can't be physically rotated, a "Flip Screen" option would be a nice enhancement.

**Comment by bnice** [comment]
Flip screen would also be good for HDMI too

**Comment by bazset** [comment]
yes i would also appreciate if you implemented Flip Screen for both CRT and HDMI.

**Comment by arbitraryfunction** [comment]
Another shout for a flip screen option. Would be amazing.
